### PR TITLE
qemu_v8: add support for EL3 SPMC

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Please wait with this until after the release.

Adds support for building TF-A and OP-TEE the SPMC at EL3.

A special manifest is added to be consumed by the SPMC.

Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>

Needed by https://github.com/OP-TEE/optee_os/pull/5141

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
Now that https://github.com/OP-TEE/optee_os/pull/5141 is merged, can we do this one too?

File: qemu_v8.mk [hunk 1]
```diff
@@ -46,7 +46,7 @@ GICV3 ?= y
 # 2:	not supported, SPMC at S-EL2 (in Hafnium), SPMD at EL3 (in TF-A)
 # 1:	SPMC at S-EL1 (in OP-TEE), SPMD at EL3 (in TF-A)
 SPMC_AT_EL ?= n
-ifneq ($(filter-out n 1,$(SPMC_AT_EL)),)
+ifneq ($(filter-out n 1 3,$(SPMC_AT_EL)),)
```
Comment (jforissier) [inline qemu_v8.mk:49]: Comment needs fixing ("3: not supported")

**Review by jforissier — COMMENTED** [review]
> Now that [OP-TEE/optee_os#5141](https://github.com/OP-TEE/optee_os/pull/5141) is merged, can we do this one too?

Sure! Apart from the missing comment update, LGTM.

`Acked-by: Jerome Forissier <jerome@forissier.org>`

**Comment by jenswi-linaro** [comment]
IBART is happy now.

**Pull request merged**